flow-typed/Collections.js

```diff
@@ -14,14 +14,21 @@ declare type CollectionState = {
   pending: CollectionGroup,
   edited: CollectionGroup,
   builtin: CollectionGroup,
-  saved: Array<string>,
```

Comment:
I'm still seeing less playlists than in production

https://odysee-workspace.slack.com/archives/C02G20Z2AEL/p1654661641103199?thread_ts=1654624401.430989&cid=C02G20Z2AEL